src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -27,10 +27,6 @@ public abstract class PlatformInstallerBase
             _logger = loggerFactory.CreateLogger(GetType());
         }
 
-        public abstract string GetInstallerScriptSnippet(string version);
```

Comment:
Made a change here since this contract is not flexible for .NET Core where version can be either runtime for sdk version.